### ISSUE TITLE
hdiutil hangs creating large images?

### ISSUE THREAD

**Issue by blezek** [create-dmg/create-dmg#108]
I can create a `500MB` `.dmg` without any problem, however, when I try to create a `.dmg` for a `2.56GB` application, the `hdiutil` hangs.  I see the `rw.*.dmg` being created, but the command doesn't finish (after an hour).

Have others seen this issue? Is there some debugging I can do?  

```
> create-dmg --version
create-dmg 1.0.8
> uname -a
Darwin XXXXX 19.6.0 Darwin Kernel Version 19.6.0: Thu Oct 29 22:56:45 PDT 2020; root:xnu-6153.141.2.2~1/RELEASE_X86_64 x86_64
```

My organization has an invasive anti-virus software installed on my Mac (not sure if that is relevant, but it's become an issue before).

Thanks for the great tool!

**Comment by infinizac** [comment]
I'm getting something quite similar to this. bump.

**Comment by pzhlkj6612** [comment]
Hi, there.

I think this issue might be fixed by #118 . The `rw.*.dmg` file is a temp file and it will not be removed due to a "resource busy" error when executing `hdiutil detach` . Please see #114 and #115 for details.

Hi, @aonez . Maybe you can release a new version?

**Comment by aonez** [comment]
Sure, let's find the time to do a release.

**Comment by aonez** [comment]
Should be fixed in 1.0.9. Thanks!!!
